tutorials/text_to_graph.ipynb

```diff
@@ -0,0 +1,192 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fiting and evaluating our recommender model\n",
+    "\n",
+    "As a recommendation algorithm integrating item network we use [Matrix Co-Factorization (MCF)](http://papers.www2017.com.au.s3-website-ap-southeast-2.amazonaws.com/proceedings/p1113.pdf), other choices are possible, e.g., [C2PF](https://www.ijcai.org/proceedings/2018/0370.pdf). We further retain [Probabilistic Matrix Factorization (PMF)](https://papers.nips.cc/paper/3208-probabilistic-matrix-factorization.pdf) as a baseline to assess the impact of the auxiliary information. Recall that, without the auxiliary information MCF reduces to PMF. To measure performance, we retain three metrics, namely Precision@50, Recall@50 and RMSE. All models and metrics are already implemented in Cornac, making such an experiment straightforward.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from cornac.eval_methods import RatioSplit\n",
+    "\n",
+    "# train/test data spliting\n",
+    "ratio_split = RatioSplit(data=ratings, test_size=0.2, exclude_unknowns=True,\n",
+    "                         item_graph=item_graph_modality, verbose=True, seed=123)\n",
+    "\n",
+    "# instantiate the recommender models\n",
+    "pmf = cornac.models.PMF(k=40, max_iter=100, learning_rate=0.001, verbose=True, seed=123)\n",
+    "mcf = cornac.models.MCF(k=40, max_iter=100, learning_rate=0.001, verbose=True, seed=123)\n",
+    "\n",
+    "# instantiate evaluation metrics\n",
+    "rmse = cornac.metrics.RMSE()\n",
+    "pre = cornac.metrics.Precision(k=50)\n",
+    "rec = cornac.metrics.Recall(k=50)\n",
+    "\n",
+    "# instantiate and run your experiment\n",
+    "exp = cornac.Experiment(eval_method=ratio_split,\n",
+    "                        models=[pmf,mcf],\n",
+    "                        metrics=[rmse, pre, rec],\n",
+    "                        user_based=True)\n",
+    "exp.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Output:**\n",
+    "<pre>\n",
+    "    |   RMSE | Precision@50 | Recall@50 | Train (s) | Test (s)\n",
+    "--- + ------ + ------------ + --------- + --------- + --------\n",
+    "PMF | 1.5082 |       0.0024 |    0.1002 |    5.6788 |   1.2293\n",
+    "MCF | 1.2823 |       0.0043 |    0.1846 |   10.1189 |   1.1811\n",
+    "</pre>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Clearly, MCF offers an important improvement over PMF. This highlights the importance of cross-modality transformations, which allowed us to take advantage of the available auxiliary data, namely item textual descriptions, while using the MCF model integrating item network into personalized recommendation.   "
```

Comment:
`a significant improvement` might sound better and avoid repetition with the word `importance` of the next sentence.